src/components/treeActivity/index.tsx

```diff
@@ -122,6 +205,18 @@ const TreeActivity: React.FC<TreeActivityProps> = ({
               <Col span={1} />
               <Col span={18}>
                 <EntryMessage>{value.message}</EntryMessage>
+                {(value.userId === currentUserId || userIsAdmin) && (
+                  <DeleteActivityButton
+                    type="primary"
+                    onClick={() => {
+                      setShowForm(!showForm);
+                      setActivityToDelete(value.activityId);
+                    }}
+                  >
+                    <DeleteOutlined />
+                    {DeleteModalDisplay()}
+                  </DeleteActivityButton>
+                )}
               </Col>
```

Comment:
⛏️ Could we be able to do this so that the text wraps around correctly (just adding a new column for the buttons)?

```suggestion
              <Col span={16}>
                <EntryMessage>{value.message}</EntryMessage>
              </Col>
              <Col span={2}>
                {(value.userId === currentUserId || userIsAdmin) && (
                  <DeleteActivityButton
                    type="primary"
                    onClick={() => {
                      setShowForm(!showForm);
                      setActivityToDelete(value.activityId);
                    }}
                  >
                    <DeleteOutlined />
                    {DeleteModalDisplay()}
                  </DeleteActivityButton>
                )}
              </Col>
```

![image](https://user-images.githubusercontent.com/33704204/213894031-cc81ed70-0686-4065-9afb-a7ec7cff646d.png)

![image](https://user-images.githubusercontent.com/33704204/213894021-b4f9bd91-371a-460d-a8cd-3ba1dcc3dddb.png)